src/audio/asrc/asrc_farrow.c

```diff
@@ -533,7 +560,7 @@ static enum asrc_error_code initialise_filter(struct comp_dev *dev,
 	/* Reset coefficients for possible exit with error. */
 	src_obj->filter_length = 0;
 	src_obj->num_filters = 0;
-	src_obj->polyphase_filters = NULL;
```

Comment:
Looks like this was never needed for anything else but to potentially find bugs where the filter is accessed uninitialized.  For that purpose it could still be valid there.